### PR TITLE
tp-qemu: fix cfg for cpuinfo_query

### PR DESCRIPTION
**Pull request by vipmike007**
As qemu-kvm ?dump is not supported in RHEL7 ,
modify config file to make it RHEL6 only.
Add new test case in cfg for RHEL7 cpuinfo query

ID: 1044928

Signed-off-by: Mike Cao <bcao@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/cpuinfo_query.cfg [hunk 1]
```diff
@@ -1,9 +1,10 @@
 - cpuinfo_query:
     type = cpuinfo_query
     vms = ''
+    start_vm = no
     variants:
         - qcpu_id:
-            no Host_RHEL.5
+            only Host_RHEL.6
```
Comment (xutian) [inline qemu/tests/cfg/cpuinfo_query.cfg]: this case not support by RHEL.7.0 host ?

File: qemu/tests/cfg/cpuinfo_query.cfg [hunk 2]
```diff
@@ -11,10 +12,13 @@
             query_cmd = " -M ?"
             cpu_info = "pc,rhel6.4.0,rhel6.3.0,rhel6.2.0,rhel6.1.0,rhel6.0.0,rhel5.5.0,rhel5.4.4,rhel5.4.0"
         - qcpu_dump:
-            no Host_RHEL.5
+            only Host_RHEL.6
```
Comment (xutian) [inline qemu/tests/cfg/cpuinfo_query.cfg]: same question with line#7

File: qemu/tests/cfg/cpuinfo_query.cfg [hunk 3]
```diff
@@ -11,10 +12,13 @@
...
             query_cmd = " -cpu ?dump"
             cpu_info = "Opteron_G4,Opteron_G3,Opteron_G2,Opteron_G1,SandyBridge,Westmere,Nehalem,Penryn,Conroe"
         - qcpu_model:
-            no Host_RHEL.5
+            only Host_RHEL.6
             query_cmd = " -cpu ?model"
```
Comment (xutian) [inline qemu/tests/cfg/cpuinfo_query.cfg]: same question with line#7

File: qemu/tests/cfg/cpuinfo_query.cfg [hunk 4]
```diff
@@ -11,10 +12,13 @@
...
             cpu_info = "Opteron_G4,Opteron_G3,Opteron_G2,Opteron_G1,SandyBridge,Westmere,Nehalem,Penryn,Conroe"
+        - qcpu_info:
+            query_cmd = " -cpu ?"
+            cpu_info = "Opteron_G4,Opteron_G3,Opteron_G2,Opteron_G1,SandyBridge,Westmere,Nehalem,Penryn,Conroe"
```
Comment (xutian) [inline qemu/tests/cfg/cpuinfo_query.cfg]: I think are your sure we will got same list in all version's qemu?

File: qemu/tests/cfg/cpuinfo_query.cfg (hunk 1 above)
Comment (vipmike007) [inline qemu/tests/cfg/cpuinfo_query.cfg]: Yes, Confirmed on qemu-kvm-rhev-1.5.3-60.el7ev.x86_64 by using #qemu-kvm -cpu ?cpu_id

File: qemu/tests/cfg/cpuinfo_query.cfg (hunk 2 above)
Comment (vipmike007) [inline qemu/tests/cfg/cpuinfo_query.cfg]: Yes, Confirmed on qemu-kvm-rhev-1.5.3-60.el7ev.x86_64 by using #qemu-kvm -cpu ?dump

File: qemu/tests/cfg/cpuinfo_query.cfg (hunk 3 above)
Comment (vipmike007) [inline qemu/tests/cfg/cpuinfo_query.cfg]: Yes, Confirmed on qemu-kvm-rhev-1.5.3-60.el7ev.x86_64 by using #qemu-kvm -cpu ?model

File: qemu/tests/cfg/cpuinfo_query.cfg (hunk 4 above)
Comment (vipmike007) [inline qemu/tests/cfg/cpuinfo_query.cfg]: the scripts works fine on latest RHEL6.7 host.
Do we still need to care about old versions RHEL6.x ? I can modify if it is needed.

Mike

**Pull request closed without merging**